js/ui/messageTray.js

```diff
@@ -1014,6 +1015,8 @@ MessageTray.prototype = {
             this._notification.actor._parent_container.remove_actor(this._notification.actor);
         }
 
+    let canShowNonCriticalInFullscreen = this._nonCriticalDisplayFullscreen;
```

Comment:
this is not aligned with the other lines

Reply:
So annoying lol, I swear it was aligned when I initially made the change. No idea what happened.